src/SFA.DAS.EAS.Web/ViewModels/TransferApprenticeshipSumaryViewModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.EmployerCommitments.Web.ViewModels
+{
+    public sealed class TransferApprenticeshipSumaryViewModel
```

Comment:
Does the name of this class reflect what it is? I'm not sure. In any case there's a typo in Summary so has to be revisited. ;-)

Reply:
Typo corrected. and I've changed the name to TransferCourseSummaryViewModel